content/tracing/go.md

```diff
@@ -8,15 +8,43 @@ customnav: tracingnav
 
 ### Installation
 
-To begin tracing applications written in Go, first [install and configure the Datadog Agent](/tracing#installing-the-agent).
+To begin tracing applications written in Go you need to:
 
-Next, install the Go Tracer from the Github repository:
+ - [Install and configure the Datadog Agent](/tracing#installing-the-agent).
+
+ -  Install the Go Tracer from the Github repository:
```

Comment:
Let's put the code in the bullet point, since it's brief:

- Get the Go tracer library: `go get "github.com/DataDog/dd-trace-go/tracer"`